portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminContentAPI/SupportContentVersioning/StructuredContentAssetPriority.testcase

```diff
@@ -55,4 +55,27 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test StructuredContentDraftContainsSetPriorityValue {
+		property portal.acceptance = "true";
+
+		task ("When create a structuredContent draft with Headless Admin Content POST '/v1.0/sites/{siteId}/structured-contents/draft' with a priority field set (as in the json example attached)") {
+			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
```

Comment:
I think we can shorten the step a bit here, for example, to `When in Headless Admin Content I create a structuredContent draft with with a priority field set`